storage/adapters/pgp_adapter.js

```diff
@@ -111,13 +105,12 @@ class PgpAdapter extends BaseAdapter {
 			.connect()
 			.then(co => {
 				connectionObject = co;
-				return Promise.resolve(true);
+				return true;
 			})
 			.finally(() => {
 				if (connectionObject) {
 					connectionObject.done();
 				}
-				return Promise.resolve(true);
```

Comment:
is connectionObject.done() equivalent to this removed line?

Reply:
Yes i read the documentation and it says return value from `finally` has no impact on resolved value. http://bluebirdjs.com/docs/api/finally.html